src/main/java/org/synyx/urlaubsverwaltung/person/api/PersonApiController.java

```diff
@@ -110,13 +111,18 @@ public ResponseEntity<PersonDto> getPerson(@PathVariable Long personId) {
     )
     @GetMapping(produces = {APPLICATION_JSON_VALUE, HAL_JSON_VALUE})
     @PreAuthorize(IS_BOSS_OR_OFFICE)
-    public ResponseEntity<PersonsDto> persons() {
+    public ResponseEntity<PersonsDto> persons(@RequestParam(defaultValue = "ACTIVE_PERSONS") PersonSearchScope scope) {
```

Comment:
We should:
* Add a fixed "value" to the `@RequestParam` to not rely on the name of the variable "scope"
* I am not sure if "scope" is a good term for this. I have to think about it.

Reply:
Also we have to adapt the documentation